src/modules/MainUi.js

```diff
@@ -3,16 +3,27 @@ import api from './Api.js';
 class MainUi {
   setup = async () => {
     await this.showList();
+    this.idApp = await api.createApp(); 
+    await this.showLikes();
```

Comment:
This approach is resetting the likes count on each page refresh, what do you think about manually creating an app key? and using that globally for all of the likes, and comments, this way we'll be able to keep the likes and comments and show them to the users.

Reply:
Yes, I think it's time to merge it.